widgetssdk/src/main/java/com/glia/widgets/call/CallController.java

```diff
@@ -601,8 +647,14 @@ public void newEngagementLoaded(OmnicoreEngagement engagement) {
         emitViewState(callState.engagementStarted(
                 engagement.getOperator().getName(),
                 operatorProfileImgUrl,
-                Utils.toMmSs(0))
+                String.valueOf(0))
```

Comment:
just "0"?

Reply:
Yeah its this view: https://www.figma.com/file/x5BLxm2UWpm3TuerdqmYi8/Glia-Mobile-UI-Components?node-id=263%3A1889